### PR TITLE
fix: renovate configuration

### PR DESCRIPTION
**Pull request by muhammedtanriverdi-tomtom**
Fixing the CocoaPods renovate configuration for the new folder structure introduced by SPM support #45

### CODE REVIEW

**Review by AlinaUshakova-TomTom — APPROVED** [review]

File: .github/workflows/run-renovate.yaml [hunk 1]
```diff
@@ -64,4 +68,4 @@ jobs:
 
       - name: Clear .netrc file
         if: always()
-        run: echo -n > ~/.netrc
+        run: echo -n > ~/.netrc
```
Comment (AlexandruLazarescu-TomTom) [inline .github/workflows/run-renovate.yaml:71]: - [ ] add new line

File: .github/workflows/run-renovate.yaml [hunk 2]
```diff
@@ -19,8 +22,9 @@ jobs:
       id-token: write
     env:
       CP_HOME_DIR: "~"
+      BUNDLE_GEMFILE: '${{ env.cocoapods_dir }}/Gemfile'
```
Comment (AlexandruLazarescu-TomTom) [inline .github/workflows/run-renovate.yaml:25]: - [ ] you can set directly here the to `CocoaPods/Gemfile` and dismiss the global env.

**Comment by AlexandruLazarescu-TomTom** [comment]
You also need to change the files taken into account by renovate:
- https://github.com/tomtom-international/tomtom-navigation-ios-examples/blob/main/.github/renovate.json5#L20
- https://github.com/tomtom-international/tomtom-navigation-ios-examples/blob/main/.github/renovate.json5#L31

**Review by AlexandruLazarescu-TomTom — CHANGES_REQUESTED** [review]
https://github.com/tomtom-international/tomtom-navigation-ios-examples/pull/48#issuecomment-1818913784

File: .github/workflows/run-renovate.yaml (hunk 2 above)
Comment (muhammedtanriverdi-tomtom) [reply .github/workflows/run-renovate.yaml]: Done!

**Review by AlexandruLazarescu-TomTom — APPROVED** [review]

**Pull request merged**